test/generation.test.js

```diff
@@ -11,13 +13,101 @@ function extractLocalesFromAcceptLanguageHeader(acceptLanguageHeader) {
     return extractedLocales;
 }
 
+const getUserAgent = (headers) => {
```

Comment:
Maybe this could be a function outside the tests, in the class file? (since you do exactly the same thing in the getOrderFromUserAgent method)

Reply:
yep, sure